### PR TITLE
Feature/generalizing methods dag

### PR DESCRIPTION
**Pull request by AlejandroMus**

### CODE REVIEW

**Review by ggsdc — COMMENTED** [review]
Will review it after the other PR is merged

**Review by ggsdc — CHANGES_REQUESTED** [review]
You should add some tests to these methods

File: libs/client/cornflow_client/core/instance_solution.py
Comment (ggsdc) [inline libs/client/cornflow_client/core/instance_solution.py:40]: This method is not getting tested

File: libs/client/cornflow_client/core/instance_solution.py
Comment (ggsdc) [inline libs/client/cornflow_client/core/instance_solution.py:412]: These methods are not getting tested

**Review by ggsdc — CHANGES_REQUESTED** [review]

**Comment by ggsdc** [comment]
Just as a good practice. All these commits where pushed together to the repo: 

image

That can be seen because just the last one is the one that launched the automated workflows. When working on your local machine, you can ammend the commit and add more changes if the commit has not been pushed to the remote, that way all those commits could be just one.

**Review by ggsdc — DISMISSED** [review]

**Comment by ggsdc** [comment]
Will wait for the other PR to be merged before merging this one

**Review by marioncottard — DISMISSED** [review]

**Review by ggsdc — APPROVED** [review]

**Pull request merged**
